src/ast/operator.rs

```diff
@@ -51,6 +51,8 @@ pub enum UnaryOperator {
     PGPrefixFactorial,
     /// Absolute value, e.g. `@ -9` (PostgreSQL-specific)
     PGAbs,
+    /// Special Not, e.g. `! false` (Hive-specific)
+    SpecialNot,
```

Comment:
```suggestion
    /// Unary logical not operator: e.g. `! false` (Hive-specific)
    BangNot,
```
Figured we can say Bang instead of Special since its conventional to call bang operator

Reply:
This comment is unadressed? (in that the PR still uses 'special not')

Reply:
@iffyio Sorry, my oversight caused the code not to be merged.